packages/iov-bcp/src/transactions.ts

```diff
@@ -219,13 +219,37 @@ export interface SendTransaction extends UnsignedTransaction {
   readonly memo?: string;
 }
 
+export type SwapTimeout = BlockHeightTimeout | TimestampTimeout;
+
+export interface BlockHeightTimeout {
+  /** Absolute block height */
+  readonly height: number;
+}
+
+export function isBlockHeightTimeout(timeout: SwapTimeout): timeout is BlockHeightTimeout {
+  return typeof (timeout as BlockHeightTimeout).height === "number";
+}
+
+export interface TimestampTimeout {
+  /** Unix timestamp in seconds */
+  readonly timestamp: number;
```

Comment:
It is implicitly specified (https://en.wikipedia.org/wiki/Unix_time) but I'll add

Reply:
hmm, when I start to add this I have to repeat all the definition of the Unix time including how leap seconds work etc.

I think it is out of scope and we just know that unix time is always a time-zone free absolute time.

Reply:
Sure, maybe add that link there, I wasn't sure myself about timezones. Or that "unix time" was so specifically defined (I just knew since jan 1, 1970)

Reply:
👍 Note to myself: improve specification of timestamp as soon as we have a resolution defined (see https://github.com/iov-one/weave/issues/447)